modules/optimizing-the-MTU-for-your-network.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// networking/network-optimization.adoc
+
+[id='optimizing-the-mtu-for-your-network-{context}']
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network
+interface card (NIC) MTU and the SDN overlay's MTU.
+
+MTUs are now autodetected, so you do not need to adjust MTU settings. When the
+Cluster Network Operator runs, it determines the MTU used by the default route,
+then sets the MTU accordingly. If you want to have a fixed MTU size for your
+workloads, then you must set the MTU explicitly, then set the network
+configuration before installation.
+
+.Procedure
+
+To set the MTU explicitly, complete the following.
+
+. Run:
++
+----
+$ openshift-installer create install-config
```

Comment:
This procedure shows `$ openshift-installer` syntax, while an earlier procedure used `$ ./openshift-installer` syntax. Guessing the earlier one can drop the `./`.